.gitlab-ci.yml

```diff
@@ -450,12 +442,8 @@ pkg:nix:deploy:channel:
 
 pkg:nix:
   extends: .nix-template
-  except:
-    refs:
-      - master
-      - /^v.*\..*$/
-    variables:
-      - $ONLY_WINDOWS == "true"
+  script:
+    - nix-build "$CI_PROJECT_URL/-/archive/$CI_COMMIT_SHA.tar.gz" -K
```

Comment:
I'm not sure I understand this line...

Reply:
`CI_PROJECT_URL` and `CI_COMMIT_SHA` are set by the GitLab CI system. With them, we build the URL of the GitLab tarball archive for the commit being tested. And we build it with `nix-build`. Given that we haven't yet switched to using the Coq Nix Toolbox in this PR, this means that this will build the legacy `default.nix`, which includes the documentation and the test-suite.

Reply:
ok thanks for the explanation...